tiledb/sm/serialization/query.cc

```diff
@@ -1076,8 +1245,9 @@ Status query_est_result_size_reader_from_capnp(
     auto result_size = it.getValue();
     est_result_sizes_map.emplace(
         name,
-        Subarray::ResultSize{
-            result_size.getSizeFixed(), result_size.getSizeVar(), 0});
+        Subarray::ResultSize{result_size.getSizeFixed(),
```

Comment:
Does this and below need to be wrapped in `hasSizeValidity` or need default values for if `SizeValidity` is missing from the cap'n proto message?

Reply:
Same thing here, leaving as-is.